src/elements/content-sidebar/activity-feed/annotation-thread/AnnotationThreadContent.js

```diff
@@ -55,39 +59,49 @@ const AnnotationThreadContent = ({
     });
 
     return (
-        <ActivityThread
-            hasReplies
-            getAvatarUrl={getAvatarUrl}
-            getMentionWithQuery={getMentionWithQuery}
-            isAlwaysExpanded
-            isRepliesLoading={isLoading}
-            mentionSelectorContacts={mentionSelectorContacts}
-            onReplyCreate={handleReplyCreate}
-            onReplyDelete={handleReplyDelete}
-            onReplyEdit={handleReplyEdit}
-            replies={replies}
-            repliesTotalCount={replies.length}
-        >
-            {error && <ActivityError {...error} />}
-            {isLoading && (
-                <div className="AnnotationThreadContent-loading" data-testid="annotation-loading">
-                    <LoadingIndicator />
-                </div>
-            )}
+        <>
+            <ActivityThread
+                hasReplies
+                getAvatarUrl={getAvatarUrl}
+                getMentionWithQuery={getMentionWithQuery}
+                getUserProfileUrl={getUserProfileUrl}
+                isAlwaysExpanded
+                isRepliesLoading={isLoading}
+                mentionSelectorContacts={mentionSelectorContacts}
+                onReplyDelete={handleReplyDelete}
+                onReplyEdit={handleReplyEdit}
+                replies={replies}
+                repliesTotalCount={replies.length}
+            >
+                {error && <ActivityError {...error} />}
+                {isLoading && (
+                    <div className="AnnotationThreadContent-loading" data-testid="annotation-loading">
+                        <LoadingIndicator />
+                    </div>
+                )}
+                {annotation && (
+                    <AnnotationActivity
+                        getAvatarUrl={getAvatarUrl}
+                        currentUser={currentUser}
+                        isCurrentVersion
+                        item={annotation}
+                        getMentionWithQuery={getMentionWithQuery}
+                        getUserProfileUrl={getUserProfileUrl}
+                        mentionSelectorContacts={mentionSelectorContacts}
+                        onEdit={handleAnnotationEdit}
+                        onDelete={handleAnnotationDelete}
+                        onStatusChange={handleAnnotationStatusChange}
+                    />
+                )}
+            </ActivityThread>
             {annotation && (
-                <AnnotationActivity
-                    getAvatarUrl={getAvatarUrl}
-                    currentUser={currentUser}
-                    isCurrentVersion
-                    item={annotation}
+                <ActivityThreadReplyForm
```

Comment:
Will we need to replicate this composition in each usage of `ActivityThread`? Should `ActivityThreadReplyForm` be used within that component, instead?

Reply:
We could potentially refactor how the `ActivityThread` is used in Sidebar. Right now the `ActivityThreadReplyForm` is built as part of `ActivityThread` and is rendered if `onReplyCreate` handler is passed.

Here the `ActivityThreadReplyForm` needs to be displayed below the `ActivityThread` as the activity thread's height needs to be limited, but the form should always be visible.

Reply:
I see. I think we could add a div around `children`, `renderButton()`, and the replies section [here](https://github.com/box/box-ui-elements/blob/master/src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.js#L113) that we can use to set the max-height and overflow values on. It's debatable, though, so not a blocking issue.